messaging-experiments.json

```diff
@@ -1146,5 +1146,488 @@
+                    "attributes": {
+                      "accesskey": "L"
+                    }
+                  }
+                },
+                "secondary": [
+                  {
+                    "action": {
+                      "type": "CANCEL"
+                    },
+                    "label": {
+                      "value": "Remind Me Later",
+                      "attributes": {
+                        "accesskey": "R"
+                      }
+                    }
+                  },
+                  {
+                    "label": {
+                      "string_id": "cfr-doorhanger-extension-never-show-recommendation"
+                    }
+                  },
+                  {
+                    "action": {
+                      "data": {
+                        "category": "general-cfrfeatures"
+                      },
+                      "type": "OPEN_PREFERENCES_PAGE"
+                    },
+                    "label": {
+                      "string_id": "cfr-doorhanger-extension-manage-settings-button"
+                    }
+                  }
+                ]
+              },
+              "category": "cfrFeatures",
+              "heading_text": "Watch videos while you browse",
+              "icon": "chrome://global/skin/media/pictureinpicture.svg",
+              "info_icon": {
+                "label": {
+                  "string_id": "cfr-doorhanger-extension-sumo-link"
+                },
+                "sumo_path": "extensionrecommendations"
+              },
+              "layout": "icon_and_message",
+              "notification_text": {
+                "string_id": "cfr-doorhanger-extension-notification"
+              },
+              "text": "Binge watch your favorite shows guilt-free with Picture-in-Picture - hover over a video and select the blue icon."
+            },
+            "frequency": {
+              "lifetime": 30,
+              "custom": [
+                {
+                  "period": 86400000,
+                  "cap": 1
+                }
+              ]
+            },
+            "targeting": "userPrefs.cfrFeatures",
+            "template": "cfr_doorhanger",
+            "trigger": {
+              "id": "openURL",
+              "params": [
+                "youtube.com",
+                "netflix.com",
+                "vimeo.com",
+                "twitch.com",
+                "dailymotion.com",
+                "hulu.com",
+                "ustream.com",
+                "metacafe.com",
+                "espn.com/watch",
+                "disneyplus.com",
+                "play.hbogo.com",
+                "primevideo.com"
+              ]
+            }
+          }
+        },
+        {
+          "slug": "treatment-d",
+          "ratio": 1,
+          "groups": [
+            "cfr"
+          ],
+          "value": {
+            "id": "PIP_CFR_INFORMATIONAL",
+            "groups": [
+              "cfr"
```

Comment:
If we add the message to the `cfr` group then production messages can prevent experiment messages from being shown.

Reply:
Switching over to "cfr-experiments".